app/src/UIComponents/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import './Header.css';
+import { Link } from "react-router-dom";
+
+function Header() {
+
+    return (
```

Comment:
All of this can be replaced with the Material UI navbar components/buttons, so we can delete all of this. The nav bar should also be a separate PR since you have it split off into it's own issue (https://github.com/SENG499-Company-4/Frontend/issues/29). Imo you can just leave the header out or blank for now, and implement in its own issue.

Reply:
^ https://mui.com/material-ui/react-app-bar/